EDMarketConnector.py

```diff
@@ -914,6 +916,9 @@ def set_labels(self):
 
     def login(self):
         """Initiate CAPI/Frontier login and set other necessary state."""
+        should_return: bool
+        new_data: Dict[str, Any] = {}
```

Comment:
```suggestion
        new_data: dict[str, Any] = {}
```

Reply:
Might be beginning to regret incorporating this typing change into my PR ;)

Reply:
Yes, tbh I'm about to merge my mypy PR, and that will have fixed/changed these things anyway.  So, when you rebase on top of that, please just drop any of your commits that are only that sort of thing and not about *your* code changes.